domains/www.dhlcgd.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dhlcgd",
+        "email": "dhlcgdofficial@gmail.com"
+    },
+    "record": {
+        "CNAME": "ghs.google.com"
```

Comment:
Provide site preview.

Reply:
My website is live at [dh-lcgd.blogspot.com](dh-lcgd.blogspot.com)

Reply:
To use Blogger, I need to point the CNAME to ghs.google.com and the other CNAME is used for Google to authenticate that I am allowed to use this domain. The site is currently not accessible via the new domain is-a.dev until the CNAME is added and Google allows me to update the new domain name to Blogger